ci/jenkins/bin/github.sh

```diff
@@ -29,6 +29,7 @@ WCCP=""
 [ "1" == "$enable_werror" ] && WERROR="--enable-werror"
 [ "1" == "$enable_debug" ] && DEBUG="--enable-debug"
 [ "1" == "$enable_wccp" ] && WCCP="--enable-wccp"
+[ "1" == "$enable_luajit" ] && WCCP="--enable-luajit"
```

Comment:
should we have a separate variable here?

Reply:
Yes we should :)